### ISSUE TITLE
Undefined symbol: _AVAudioSession when trying to build

### ISSUE THREAD

**Issue by KeKam** [react-native-webrtc/react-native-callkeep#513]
# Bug report

- [X] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

- Reproduced on:
 - [ ] Android
 - [X] iOS

## Description
When trying to build for iOS with Xcode 13.2.1 it fails with 

## Versions
    - Callkeep:
    - React Native:
    - iOS:
    - Android:
    - Phone model: 

## Logs

```
Paste here
```

**Comment by KeKam** [comment]
Any info regarding this ?

**Comment by Romick2005** [comment]
I am using my custom version in fork: 
in package.json    
```"react-native-callkeep": "git+https://github.com/Romick2005/react-native-callkeep.git",``` and do not have such issue.

**Comment by KeKam** [comment]
Yes `#import "RNCallKeep.h"` is added in AppDelegate.m. The library works but in order to be able to build it with XCode I have to manually add the `AVFAudio.framework`.

**Comment by KeKam** [comment]
I'm also not using any fork.

**Comment by 1stblood** [comment]
hi im still having this issue. adding AVFAudio.framework doesnt solved the issues. Even fork form Romick2005 doesnt solved.

Im using RN Firebase with use_frameworks!

**Comment by 1stblood** [comment]
> hi im still having this issue. adding AVFAudio.framework doesnt solved the issues. Even fork form Romick2005 doesnt solved.
> 
> Im using RN Firebase with use_frameworks!

got solved in Podfile insert this

pre_install do |installer|
  installer.pod_targets.each do |pod|
    if pod.name.eql?('RNCallKeep')
      def pod.build_type
        Pod::BuildType.static_library
      end
    end
  end
end

**Comment by elsieej** [comment]
@1stblood i spent 3 days for research how to found solution, tks you so much becuz save my life.

**Comment by robertcoroianu** [comment]
> AVFAudio.framework

Hi,
   How I can add the AVFAudio.framework? I've just started to develop first time on ios.
